modules/system/console/OctoberUtil.php

```diff
@@ -294,7 +295,57 @@ protected function utilPurgeUploads()
             return;
         }
 
-        // @todo
+        $totalCount = 0;
+        $uploadsPath = Config::get('filesystems.disks.local.root', storage_path('app')) . '/uploads';
```

Comment:
@bennothommo just an idea, maybe it would be useful to take `uploads` path from configs?

Reply:
@w20k Good idea, will change it around soon.

Reply:
@LukeTowers not yet, still on the to-do list.